### DISCUSSION TITLE
JavaScript Israel Talk

### DISCUSSION THREAD

**Discussion by benjamingr** [swc-project/swc#1460] [Show and tell]
Hey :) 

I'm going to give a talk on modern build infrastructure and I want to mention and talk about swc. The audience is usually ±100-200 people and there are ±500-1000 more views later on. The talks are mostly in Hebrew.

It's an online event:

I was wondering if (in order of priority):
 - You'd be willing to record 2-3 minutes describing the main selling points of swc (it's fast, it's trusted by deno, you can run swc with the bundler or jest easily and drop them instead of ts-loader or ts-node).
 - If not - if you'd be willing to review 2-3 minutes of me talking about it.
 - If not - if you'd be willing to review a bit of text I'll write about it.

If any of this is interesting or relevant to you - I'm available at benjamingr@gmail.com or you can comment here.

**Comment by kdy1** [comment]
Second option seems good.
I'm far from a good speaker so 1 is too hard to me.

If you want, mail me :)

**Reply by benjamingr** [reply]
I will :)

**Comment by kdy1** [comment]
And thanks!
